test/parallel/test-dns.js

```diff
@@ -239,6 +239,15 @@ assert.throws(() => dns.lookup('nodejs.org', 4), {
   name: 'TypeError'
 });
 
+assert.throws(() => dns.lookup('', {
+  family: 'nodejs.org',
+  hints: dns.ADDRCONFIG | dns.V4MAPPED | dns.ALL,
+}), {
+  code: 'ERR_INVALID_ARG_TYPE',
+  name: 'TypeError',
+  message: 'The "callback" argument must be of type function. Received undefined'
```

Comment:
There is no need to check the `message`. Can you remove this line?

Reply:
Error message's tend to change from time to time. To reduce the maintenance costs and reduce unrelated errors, we never test the error message, but always the error code.

Reply:
Sure, I will do that